app/models/user.rb

```diff
@@ -82,6 +82,9 @@ class User < ApplicationRecord
 
   validates :email, presence: true
 
+  ROLES = %w[challenge_manager evaluator solver].freeze
+  validates :role, inclusion: { in: ROLES }
```

Comment:
on the phx side there are a few admin roles too, shouldn't we include those roles too? atm I'm not sure what those roles can do in the Rails app yet.